net/lighttpd/Makefile

```diff
@@ -57,6 +57,11 @@ PKG_BUILD_DEPENDS:= \
   LIGHTTPD_CRYPTOLIB_MBEDTLS:mbedtls \
   LIGHTTPD_CRYPTOLIB_WOLFSSL:wolfssl
 
+cryptolibdep= \
```

Comment:
I think cryptolib and cryptolibdep can be simply removed,
crypto dependency is defined in lighttpd already, there's no need to add it to submodules again

Reply:
I'd like to keep the dependencies declared unless you think that will cause a problem.

It would be minimalist to remove the declared dependencies due to the transitive dependency on the crypto library via dependency on lighttpd which depends on the crypto library.

However, each module which depends on crypto libs does so directly.  The lighttpd base depends on crypto libs for cryptographic randomness in lighttpd `src/rand.c`, but other modules depending on crypto libs do so for digest functions such as MD5, SHA1, SHA256, SHA512, HMAC.

That said, I will adjust the patch to check more directly for dependency on nettle for build flag `-Dwith_nettle=true`